README.md

```diff
@@ -31,63 +42,153 @@ envs:
 - A_SECRET_PARAM_TWO: the value for secret two
 ```
 
-## How to create your own step
+## Simple usage
 
-1. Create a new git repository for your step (**don't fork** the *step template*, create a *new* repository)
-2. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-3. Fill the `step.sh` with your functionality
-4. Wire out your inputs to `step.yml` (`inputs` section)
-5. Fill out the other parts of the `step.yml` too
-6. Provide test values for the inputs in the `bitrise.yml`
-7. Run your step with `bitrise run test` - if it works, you're ready
+### Example using public IDs
 
-__For Step development guidelines & best practices__ check this documentation: [https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md).
+<!-- TODO: change git urls to step references after we publish it -->
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - public_ids: '012-345-678, 9ab-cde-fff'
+```
+
+### Example task using existing `synthetics.json` files
+
+```yaml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - files: 'e2e-tests/*.synthetics.json'
+```
 
-**NOTE:**
+For an example test file, see this [`test.synthetics.json` file][C-2].
 
-If you want to use your step in your project's `bitrise.yml`:
+## Complex usage
 
-1. git push the step into it's repository
-2. reference it in your `bitrise.yml` with the `git::PUBLIC-GIT-CLONE-URL@BRANCH` step reference style:
+### Example task using the `testSearchQuery`
 
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - test_search_query: 'tag:e2e-tests'
 ```
-- git::https://github.com/user/my-step.git@branch:
-   title: My step
+
+### Example task using the `testSearchQuery` and variable overrides
+
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
    inputs:
-   - my_input_1: "my value 1"
-   - my_input_2: "my value 2"
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - test_search_query: 'tag:e2e-tests'
+   - variables: |
+      START_URL=https://staging.website.com
+      PASSWORD=$(StagingPassword)
```

Comment:
I doubt that `$(StagingPassword)` is the actual syntax to refer to secrets in Bitrise 🤔 

Also the `StagingPassword` camel case is really "Microsoft-y" 😅 

Something like this would probably work:

```suggestion
      PASSWORD=$STAGING_PASSWORD
```